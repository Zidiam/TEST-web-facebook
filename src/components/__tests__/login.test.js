import {render, screen, cleanup} from '@testing-library/react';
import renderer from 'react-test-renderer';
import LoginButton from '../loginButton';
afterEach(() =>{
    cleanup();
});

test('should render Login Button', () => {
    render(<LoginButton/>);
    const loginElement = screen.getByTestId('login-button');
    expect(loginElement).toBeInTheDocument();
    expect(loginElement).toHaveTextContent('Login');
    //expect(loginElement).not.toContainHTML('strike');
});

test('matches snapshot', () =>{
    const tree = renderer.create(<LoginButton/>).toJSON();
    expect(tree).toMatchSnapshot();
});