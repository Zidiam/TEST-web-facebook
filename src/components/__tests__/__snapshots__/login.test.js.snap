// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<button
  data-testid="login-button"
  id="login-button"
>
  Login
</button>
`;
